client/components/NotificationModal.jsx

```diff
@@ -42,3 +69,5 @@ NotificationModal.propTypes = {
         action: PropTypes.func,
     }),
 };
+
+export const NotificationModal = connect(null, null)(NotificationComponent);
```

Comment:
Using `connect` here should not be required (as `null, null` is being passed through).
You should be able to use `export class NotificationModal` instead

Reply:
Done